src/one_dragon_qt/widgets/zoomable_image_label.py

```
@@ -1,55 +1,298 @@
+        """
+        设置图像的接口，兼容Cv2Image和QPixmap
+        :param image: 图像对象
+        :param preserve_state: 是否保留当前的缩放和位置状态
+        """
+        if image is None:
+            self.original_pixmap = None
+            self.current_scaled_pixmap = QPixmap()
+            self.update()
+            return
+
+        # 如果是Cv2Image对象，获取其QPixmap
```

Ruff (0.12.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)